components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -654,6 +662,10 @@ components:
         className:
           type: string
           example: org.wso2.carbon.user.core.jdbc.UniqueIDJDBCUserStoreManager
+        isLocalUserStore:
```

Comment:
since userstore part is anyway implied from the API context, shall we rename this to "isLocal"?

Reply:
Fixed